implementations/code/code_changes_git.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+from itertools import groupby
+import conditions
+import utils
+from commit_git import CommitGit
+
+
+class CodeChangesGit(CommitGit):
+    """
+    Class for Code_Changes for Git repositories.
+    """
+
+    def compute(self):
+        """
+        Count number of commits of different types, like including
+        empty commits or counting only those commits made on
+        the master branch.
+
+        :returns count: the number of commits satisfying the conditions passed
+            while instantiating CodeChangesGit.
+        """
+
+        commit_hashes = [item['hash'] for item in self.items]
```

Comment:
To avoid this, you can either make commit_hashes directly a set, or make it a dictionary, and then just count the dictionary (adding an existing key to a dictionary does not add a new element)

Reply:
Ok. Used a set comprehension to directly make `commit_hashes` a set.